docs/blog/2019-04-03-how-we-think-about-product-at-gatsby/index.md

```diff
@@ -0,0 +1,130 @@
+
+Instead of giving this outright, we instead look at the problem we’re trying to solve, do prototyping and feasibility testing, and create a scope. After taking the time to understand what the problem is, how we can solve it, and how much time it will take us to create the solution, we’re able to give a much better estimate for highly awaited products or features.
+
+We won’t give these high-integrity commitments for everything, but if we know that sales or marketing is waiting on us to move forward, then we have a reasonable timeframe to talk about that’s based in reality.
+
+## How does the product organization fit in with the rest of the company?
+
+The product org is the group of people within Gatsby responsible for identifying product opportunities, doing customer discovery, usability research, and design. That means the product organization needs to understand the needs of the customer as well as the needs of the company in order to make sure that both are successful.
+
+## What processes do we want to focus on?
+
+Since we are a growing startup, any process we write here will probably change a bit in the next few months. What won’t change is how we think about product and what we communicate across the company. That means a few key areas will be really important for us to focus on.
+
+### How we communicate product ideas
+
+We are a remote company, so it’s very important that we have common knowledge and understanding. Introducing a communication strategy that is more agile and allows for open communication is one way we will tackle this.
+That means that product initiatives will collaborate with research, engineering, and design right from the beginning to make sure we have shared context where it counts most.
+We’ll also focus on product discovery, rapid prototyping, and validation so that we can prioritize effectively. We’ll make sure to do MVPs, described as Minimum Viable _Prototypes_ in Marty Cagan’s book _Inspired_. This is really a difference in functionality and fidelity. A product will typically have many features at high fidelity, while a prototype will have limited functionality at a lower fidelity.
+We typically think of MVPs and Minimum Viable Products, but that often means that we are creating something that’s very well thought out and crafted _before_ we check to make sure it’s something people want or something we should spend our time doing. By focusing on research, discovery, and _prototyping_ we keep the focus on our end users and retain the ability to rapidly test potential solutions.
+
+### How we focus on personas
+
+Because we want to stay user-focused, we will also be doing discovery and usability research against specific personas that are important to Gatsby. As we learn more about our audience, we’ll get a better understanding of current personas and start to see new personas that we should define.
+How we prioritize
```

Comment:
```suggestion

### How we prioritize

```

Reply:
Added this heading in a separate commit. Thanks!

Reply:
Did that commit get pushed @marisamorby?